### ISSUE TITLE
How print registered details, without stacktrace

### ISSUE THREAD

**Issue by mimol91** [ansel1/merry#23]
Would it be possible to add method to print error and registered details, but without stack tracke?

**Comment by ansel1** [comment]
There are a lot of different ways users may want to print errors...how about I add a way to extract the raw details as a map, then you can craft your own version of Print()?

**Comment by mimol91** [comment]
Yeah, sounds great (I can see there are already separate methods for stacktrace and cause)

**Comment by ansel1** [comment]
Published 1.6.0 and 2.0.0-beta.10 with `RegisteredDetails(err) map`
